README.md

```diff
@@ -90,6 +90,10 @@ package-name/
 
 4. Write Tests and Publish!
 
+## Contributing
+The main purpose of this repository is to continue to evolve Lerna starter kit.
+Lerna starter kit is to enable you to have multiple React components published as separate packages under a single repo.
+Contribute guide
```

Comment:
Looking good, just one small change. 

Could you replace the line `Contribute guide` with something along the lines of: 

> Please feel free to contribute.

and will be happy to merge then.